packages/core/src/server.ts

```diff
@@ -1014,14 +1028,24 @@ export class Controller extends API {
             members,
             groups,
             vaults,
+            directory,
         });
 
-        if (org.owner !== owner) {
-            await this.provisioner.orgOwnerChanged(
-                org,
-                org.getMember({ id: org.owner })!,
-                org.getMember({ id: owner })!
-            );
+        if (org.directory.syncProvider === "scim") {
+            if (!org.directory.scim) {
+                org.directory.scim = new ScimSettings();
+                org.directory.scim.secret = await getCryptoProvider().randomBytes(16);
+                // TODO: This works locally for now, but needs to be updated to be more bullet-proof
+                org.directory.scim.url = this.config.clientUrl.replace("8080", "5000");
```

Comment:
@MaKleSoft Why wouldn't this work until I made the `url: string` default to `""` in `ScimSettings`? It would be set but not stored.